go.mod

```diff
@@ -1,39 +1,43 @@
 module github.com/cloudwan/gohan
 
 require (
+	github.com/Bowery/prompt v0.0.0-20190916142128-fa8279994f75 // indirect
```

Comment:
I see no reason to update `go.mod` (and `go.sum`) now. Moreover, some of added dependencies are removed after executing `go mod tidy` (`go1.13 linux/amd64`).

Please note that when dependencies in `gohan` are updated, other projects using `gohan` must be updated as well (go require packages to have exactly the same version in plugin and main program).

Reply:
Thanks. Will not add go.mod and go.sum .